web/packages/plugins/analysis-platform/src/modules/Asset/AssetOverview.tsx

```diff
@@ -15,6 +15,7 @@ import { DEFAULT_DATASOURCE_ID } from '../../const'
 
 export const AssetOverview = (props: { settings: DmtSettings }): ReactNode => {
   const { settings } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore